corehq/apps/app_execution/views.py

```diff
@@ -147,24 +151,31 @@ def _get_context(request, title, url, add_parent=False, **kwargs):
 
 @require_superuser_or_contractor
 @use_bootstrap5
+@use_nvd3
 def workflow_log_list(request, pk):
-    logs = AppExecutionLog.objects.filter(workflow_id=pk).order_by("-started").all()
+    utcnow = datetime.utcnow()
+    chart_data = get_avg_duration_data(
+        workflow_id=pk, start=utcnow - relativedelta(months=1), end=utcnow
+    )
     context = _get_context(
         request,
         "Automatically Executed App Workflow Logs",
         reverse("app_execution:workflow_logs", args=[pk]),
         add_parent=True,
         workflow=AppWorkflowConfig.objects.get(id=pk),
-        logs=logs,
-        total=logs.count(),
+        total=AppExecutionLog.objects.filter(workflow_id=pk).count(),
+        chart_data=chart_data
     )
     return render(request, "app_execution/workflow_log_list.html", context)
 
 
 @require_superuser_or_contractor
 @use_bootstrap5
 def workflow_logs_json(request, pk):
-    logs = AppExecutionLog.objects.filter(workflow_id=pk).order_by("-started").all()
+    limit = int(request.GET.get('per_page', 10))
+    page = int(request.GET.get('page', 1))
```

Comment:
is nvd3 1-indexed?

Reply:
This isn't nvd3 related. It comes from [pagniation.js](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/components/pagination.js#L46) which is 1-indexed